plugins/de.cognicrypt.codegenerator/src/main/java/de/cognicrypt/codegenerator/crysl/templates/stringhashing/StringHasher.java

```diff
@@ -16,20 +16,22 @@
 
 import de.cognicrypt.codegenerator.crysl.CrySLCodeGenerator;
 
+
 public class StringHasher {
 
+	
 	public static java.lang.String createHash(java.lang.String msg) throws GeneralSecurityException {
 		byte[] plainBytes = msg.getBytes(StandardCharsets.UTF_8);
 		byte[] out = null;
 		CrySLCodeGenerator.getInstance().includeClass("java.security.MessageDigest").addParameter(out, "out").generate();
 		return Base64.getEncoder().encodeToString(out);
 	}
 
-	public static boolean verifyHash(java.lang.String compareeHash, java.lang.String newMsg) {
+	public static boolean verifyHash(java.lang.String compareHash, java.lang.String newMsg) {

```

Comment:
I would also suggest adding Javadocs in both methods in this class, `createHash()` and `verifyHash()`, similar to how it is done in other templates.

Reply:
same thing here.